An upstream patch to fix ksh test issues, which also affect bash 4.0

http://bugs.gentoo.org/263068
http://git.savannah.gnu.org/cgit/automake.git/commit/?h=branch-1-10&id=319305db77dfdcbba3d5a1703c781750affdb20a

2008-12-21 Ralf Wildenhues <Ralf.Wildenhues@gmx.de>

 Testsuite fix for ksh.
 * tests/check10.test: Add ':' as last command in subshell, for
 zero exit status of the subshell.
 Missing backport from master reported by Tim Rice.

--- a/tests/check10.test
+++ b/tests/check10.test
@@ -72,6 +72,7 @@
   env TESTS="xpass xpass2" $MAKE -e check
   env TESTS='pass skip xfail' $MAKE -e check
   $MAKE check
+  :
 ) >stdout
 cat stdout
 
